docs/installation/server-config/security.rst

```diff
@@ -82,7 +82,7 @@ by adding the following to your KSQL server config.
 
 .. code:: bash
     ksql.schema.registry.basic.auth.credentials.source=USER_INFO
-    ksql.schema.registry.basic.auth.user.info=username:password
+    ksql.schema.registry.schema.registry.basic.auth.user.info=username:password
```

Comment:
really? seems odd that the one above doesn't follow the same pattern??

Reply:
Yes it is. There is a ticket to fix this inconsistency, and we may have to update this again if people agree to fix this inconsistency in schema registry within the 5.0 timeframe.

But as it stands, those are the settings you need to use to configure basic auth for Schema Registry.